src/NCronJob/Registry/JobState.cs

```diff
@@ -9,12 +9,15 @@ internal readonly struct JobState
     public DateTimeOffset Timestamp { get; }
     public Exception? Fault { get; }
 
-    public JobState(JobStateType type, Exception? fault = default)
+    public JobState(
+        JobStateType type,
+        DateTimeOffset now,
```

Comment:
Small nitpick: We should call it `utcNow`. I know it is `DateTimeOffset` so it doesn't matter so much, but I would prefer the consistency.

Reply:
Fixed